app/main.py

```diff
@@ -2,0 +1,9 @@
+def copy_file(command: str) -> None:
+    cp, old_file, new_file = command.split()
+
+    if cp != "cp":
+        raise ValueError("The command need to start with 'cp'")
+
+    if old_file != new_file:
+        with open(old_file, "r") as file_in, open(new_file, "w") as file_out:
```

Comment:
Code Style Issue: According to the checklist, you should use descriptive and correct variable names. Instead of 'file_in' and 'file_out', consider using 'source_file' and 'destination_file'.